pages/define-user-page.ts

```diff
@@ -0,0 +1,38 @@
+// Define user or root password before installation
+
+export class DefineUserPage {
+  readonly page: Page;
+  readonly pwd: Locator;
```

Comment:
let's use password instead of `pwd` to avoid this patterns of strange abreviations.

Reply:
password is also in kind of a use by the framework. Password is acceptable

Reply:
ok